test/box-luatest/gh_7846_crash_on_replica_id_alter_test.lua

```diff
@@ -0,0 +1,57 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local urilib = require('uri')
+local fiber = require('fiber')
+
+local g = t.group('before_replace alter')
+
+g.test_replica_id_inc = function()
```

Comment:
This and `test_replica_id_nullify` are almost identical. You could make them one function which would take the trigger's full body or a flag whether it should delete or replace the ID. And then call that one function 2 times. Would be ~x2 times shorter.

Reply:
Done